### ISSUE TITLE
Lecture Note 3 quadratic function NAN

### ISSUE THREAD

**Issue by tfnewbie4** [chiphuyen/tf-stanford-tutorials#22]
Hi,
I'm new to TF and came across your excellent course.

I was trying to perform the quadratic function in  lecture 3, but I get NaN for my Epochs when I run.

Is the quadratic function solution / code avail anywhere?

Attached are the changes I made in line w/ the lecture notes but I think more changes are necessary than what is in the lecture notes.  From the quadratic function plot, I see that the result is not a line but a series of X, Y plots (presumably Y_predicted) but I don't know how to get that value from TF (as of yet, that's why I'm going through this excellent course! :)

[03_linear_regression_quad.txt](https://github.com/chiphuyen/tf-stanford-tutorials/files/1063246/03_linear_regression_quad.txt)

Thanks!

**Comment by lkhphuc** [comment]
Reducing the learning rate (should be around 1e-5) solves the problem.

**Comment by msgrasser** [comment]
Yes, thank you @lkhphuc! Even at the suggested 1e-5 learning rate, the gradient descent is not converging (i.e. it is jumping too far across one of the loss's minima and actually ascending), but this is super helpful. 

Curious that tf doesn't do a great job of letting you know why optimization is breaking (though this seems it should be straightforward to identify?), so I hope to see this included in the next iteration of these great course notes!

**Comment by chiphuyen** [comment]
Like what @lkhphuc, you can reduce the learning rate. I'm closing the issue for now. Let me know if you have any other question.
